app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,10 @@
 <resources>
     <string name="app_name">SoundClound_03</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="home">home</string>
+    <string name="playlist">Playlist</string>
+    <string name="local">Local</string>
```

Comment:
3.2.2 Strings
String names start with a prefix. For example registration_email_hint or registration_name_hint. If a string doesn't belong to any section, follow rules here:
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md